notNeededPackages.json

```diff
@@ -4576,8 +4576,8 @@
             "asOfVersion": "3.0.0"
         },
         "rdf-js": {
-            "libraryName": "rdf-js",
-            "asOfVersion": "4.0.2"
+            "libraryName": "@rdfjs/types",
+            "asOfVersion": "1.0.0"
```

Comment:
not sure about this change. `@rdfjs/types` was never on DT.
The package change happened back in 2020:
https://github.com/rdfjs/types/commit/f97793d9e1ff42ad4113c687dd6996633499eb73
Should it stay as is?

Reply:
Yes, this file is used specifically for pushing out shim packages and deprecations; this change isn't correct.